libs/db.lua

```diff
@@ -289,6 +289,10 @@ return function (path)
 
       for entry in assert(fs.scandir(path)) do
         local fullPath = pathJoin(path, entry.name)
+        if entry.type == nil then
+          local stat = fs.stat(fullPath)
+          entry.type = stat.type
+	    end
```

Comment:
You are right. Thanks for the feedback Tim. I have re-based and committed again.